### ISSUE TITLE
Easier method of resizing the logo

### ISSUE THREAD

**Issue by KyleMaas** [mediacms-io/mediacms#541]
**Describe the feature you'd like**
It would be really nice if there was an easier way to increase the size of the logo.  There are two variables for it in the CSS - `logo-height` and `default-logo-height`.

https://github.com/mediacms-io/mediacms/blob/main/frontend/src/static/css/config/index.scss

But the only way I could find to actually change these values (short of editing everywhere that the variables were used in the static version) was to edit this file, run `npm install` and `npm run dist` on the frontend, and copy over the CSS.

**Describe alternatives you've considered**
I'm too new to this codebase to be able to come up with alternatives.  The method above seemed like the cleanest way to do it given what I've seen of it.

**Comment by mgogoulos** [comment]
this is valid

**Comment by Arunakjv** [comment]
Dear, Please anyone tell me the procedure of logo changing. 
i did myself but is size decreased...

**Comment by KyleMaas** [comment]
@Arunakjv Yeah, that's the problem I ran into that resulted in this feature request.  I had to patch the CSS template and regenerate it.  That's one of the main reasons why we need an easier way to do that.

**Comment by Arunakjv** [comment]
Thanks,
How can I implement the logo in my site. please tell me the procedure.
Thanks & Regards,

*Arun Kumar*
Mob: 999187841
